node/pkg/p2p/p2p.go

```diff
@@ -60,6 +60,9 @@ var heartbeatMessagePrefix = []byte("heartbeat|")
 
 var signedObservationRequestPrefix = []byte("signed_observation_request|")
 
+// Heartbeats that are this old or this much into the future will be dropped. This value should encompass clock skew and network delay.
```

Comment:
godoc strings should start with the variable name such as "heartbeatMaxTimeDifference indicates the time [...]"